tests/light/src/axosyslog_light/syslog_ng_config/statements/sources/network_source.py

```diff
@@ -61,11 +61,11 @@ def __init__(
         self.driver_name = "network"
         super(NetworkSource, self).__init__(stats_handler, prometheus_stats_handler, options=options)
 
-    def write_log(self, message, rate=None, transport=None, framed=None):
-        self.io.write_messages([message], rate=rate, transport=transport, framed=framed)
+    def write_log(self, message, rate=None, transport=None, framed=None, rate_burst_start=False):
```

Comment:
I think we does not need to use `rate_burst_start` parameter in case of 1 message which is the case in `write_log()` .

Reply:
I've removed it together with `rate`.